cmd/asset-syncer/utils.go

```diff
@@ -429,9 +430,21 @@ func (f *fileImporter) fetchAndImportFiles(name string, r *models.RepoInternal,
 
 	tarf := tar.NewReader(gzf)
 
-	readmeFileName := name + "/README.md"
-	valuesFileName := name + "/values.yaml"
-	schemaFileName := name + "/values.schema.json"
+	// decode escaped characters
+	// ie., "foo%2Fbar" should return "foo/bar"
+	decodedName, err := url.PathUnescape(name)
+	if err != nil {
+		log.Errorf("Cannot decode %s", name)
+		return err
+	}
+
+	// get last part of the name
+	// ie., "foo/bar" should return "bar"
+	fixedName := path.Base(decodedName) // get
```

Comment:
Not sure what the inline comment at the end of this line is for? Perhaps left over after moving it to above?